recipes/inja/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "inja" %}
+{% set version = "3.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/pantor/inja/archive/v{{ version }}.tar.gz
+  sha256: f4210493e7e3c62d3050ca6e5e9dd72823ec4125a469fa9b28519d7f32fc9731
+  patches:
+    - patches/0001-Do-not-install-nlohmann-json-if-it-is-not-embedded-1.patch
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - cmake
+    - make  # [unix]
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - nlohmann_json >=3.8.0
+
+test:
+  commands:
+    - test -d ${PREFIX}/include/inja  # [unix]
+    - test -f ${PREFIX}/include/inja/inja.hpp  # [unix]
+    - test -f ${PREFIX}/include/inja/renderer.hpp  # [unix]
```

Comment:
The library is header only, hence we don't check for linkages or a dynamic library.